CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Unreleased
 
 ### Added
+- Added timestamp to generate-roblox-std

```

Comment:
```suggestion
- Added timestamp to generated Roblox standard library
```

This is used outside of generate-roblox-std itself, so this is me clear. Last critique, I promise.